tests/config/test_data.config

```
@@ -0,0 +1,16 @@
+def test_data_dir = "${launchDir}/tests/data/"
+def nf_core_modules_data = "https://raw.githubusercontent.com/nf-core/modules/master/tests/data/"
```

```suggestion
def nf_core_modules_data = "https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/"
```

Forgot this got updated :grimacing: